### DISCUSSION TITLE
Polling display for min and max brightness values

### DISCUSSION THREAD

**Discussion by robert-d-94** [MonitorControl/MonitorControl#1292] [General]
I have a Dell AW3821DW which does not use the default 0-100 brightness control range. Is there a way to poll the display to figure out what I need to set my custom min and max values to? I have searched the UI and could not find a way to do this. Thanks in advance for assistance.

**Comment by waydabber** [comment]
Nope. This was present in older versions of the app but was generally unreliable. The simplest way is to try changing the max value until you see no change in the display's OSD (or with your eyes) in brightness level.
